GeoUK/OSTN02Transform.cs

```diff
@@ -1,110 +0,0 @@
-﻿using GeoUK.Coordinates;
-using GeoUK.Ellipsoids;
-using GeoUK.Projections;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Reflection;
-
-namespace GeoUK
-{
-    public static class OSTN02Transform
```

Comment:
@awelburn Dont want to remove currently public apis, can you re-add please.

What was your reasoning for removing it?

Reply:
The following files that where removing will need to be readded for this - 

- OSTN02_OSGM02_GB.txt
- ResourceManager.cs